d3ploy/tester.py

```diff
@@ -218,44 +218,54 @@ def supply_demand_dict_nond3ploy(sqlite, commod, demand_eq=0):
     return all_dict
 
 
-def residuals(all_dict):
-    """ Conducts a chi2 goodness of fit test
+def residuals_under(all_dict):
```

Comment:
would "cumulative_undersupply" be a more informative name?

Reply:
yes